euclidlib/photo/_le3_2pcf_wl.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from os import PathLike
+from typing import TYPE_CHECKING, Tuple, Union
+from numpy.typing import NDArray
+
+from euclidlib.photo import photo_data
+
+if TYPE_CHECKING:
+    from collections.abc import Sequence
+    from typing import Any, TypeAlias
+
+
+# type alias
+_DictKey: "TypeAlias" = Union[str, int, Tuple["_DictKey", ...]]
+
+def xi_tpcf(
+    path: str | PathLike[str],
+    *,
+    include: Sequence[tuple[Any, ...]] | None = None,
+    exclude: Sequence[tuple[Any, ...]] | None = None,
+) -> dict[_DictKey, NDArray[Any]]:
```

Comment:
This annotation only matches if we are actually returning values in the `_DictKey` format, see below.

Reply:
I'm sorry, I'm not familiar with the `_DictKey` format and could not really find documentation of it. Could you please tell me where to read about it or how to change my dictionary?